stdlib/jwt/src/main/ballerina/src/jwt/inbound_jwt_auth_provider.bal

```diff
@@ -38,10 +38,14 @@ public type InboundJwtAuthProvider object {
         self.inboundJwtCache = jwtValidatorConfig.jwtCache;
     }
 
-    # Authenticate with a JWT token.
-    #
-    # + credential - Jwt token extracted from the authentication header
-    # + return - `true` if authentication is successful, othewise `false` or `auth:Error` occurred during JWT validation
+# Authenticates with a JWT.
```

Comment:
```suggestion
# Authenticates with a JWT token.
```

Reply:
JWT stands for JSON Web Token therefore we don't to say token again.
@praneesha WDYT?

Reply:
+1. No need to add token.

Reply:
+1